get_started_with_gx_cloud/workshop.md

```diff
@@ -38,12 +38,28 @@ You'll need to setup a data source from GX Cloud. We provide demo data in a Post
 > **Select the `nyc_taxi_data` Data Asset**
 > 1. On the **Select tables to import** page, check the box next to `nyc_taxi_data`.
 > 1. Click **Add 1 Asset**.
+> 1. Click **Start monitoring**.
 
 <img src="img/add_demo_data_asset.png" alt="Add a Demo Data Asset" style="width:600px;"/><br>
+<img src="img/add_demo_data_monitoring.png" alt="Start Monitoring" style="width:600px;"/><br>
```

Comment:
This alt text repeats the button text that's already referenced in the preceding prose. Here's a suggestion that communicates more of what's visually available in the screenshot

```suggestion
<img src="img/add_demo_data_monitoring.png" alt="By default, GX Cloud automatically generates Expectations to detect schema and volume issues. You can de-select recommendations you’d like to opt out of." style="width:600px;"/><br>
```